src/test_wasm_simd.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+"""
+Tests that SIMD operations are compiled to intrinsic instructions.
+For each operation and vector type listed in the test cases, a
+simple test program is generated and compiled, and the compiled
+output is verified to consist of a single SIMD instruction.
+"""
+
+from __future__ import print_function
+import os
+import re
+import subprocess
+import sys
+import tempfile
+
+wasm_simd_test_cases = [
+  # Oper Arity ArgType     ReturnType   Opcode   KnownFailure
+  ('add',  2, 'int32x4',   'int32x4',   'add',   False),  # noqa
```

Comment:
`noqa` seems like a pretty big hammer, but I don't know off-hand what would be better. This is probably fine.